internal/claims/claims.go

```diff
@@ -1,7 +1,7 @@
 package claims
 
 type Custom struct {
-	Name   string   `json:"name" validate:"required"`
```

Comment:
We already had code that checked if `Name == ""`. "required" is a really easy thing to validate with an `if` statement. I don't think we need to involve any validation libraries here. 

So I removed the `validate` tag and left the existing `if` check. I did however move it into a different function.